atm.js

```diff
@@ -1,3 +1,137 @@
+// Deposit - Checking
+$('#checking').find($('.deposit')).on('click',depositChecking);
+function depositChecking(){
+  var depositAmount = parseInt($('#checking').find($('.input')).val());
+  if(checkingBalance<0){
+    overdraftCheck(depositAmount);
+  }
+  else{
+    checkingBalance = checkingBalance + depositAmount;
+    $('#checking').find($('.balance')).html('$' + checkingBalance);
+    alert('Thank you.  GNB loves you!');
+  }
+};
+
+// Withdraw - Checking
+$('#checking').find($('.withdraw')).on('click',withdrawChecking);
+function withdrawChecking(){
+  var withdrawAmount = parseInt($('#checking').find($('.input')).val());
+  checkingBalance = checkingBalance - withdrawAmount;
+  if(checkingBalance<0){
+    overdraftApproval();
+  }
+  else if(checkingBalance=0){
+    $('#checking').find($('.balance')).html('$' + checkingBalance);
+    $('#checking').find($('.balance')).css('background-color','red');
+    alert('Thank you.  Please take your cash.  And remember, GNB loves you!');
+  }
+  else{
+    $('#checking').find($('.balance')).html('$' + checkingBalance);
+    alert('Thank you.  Please take your cash.  And remember, GNB loves you!');
+  }
+};
+
+// Deposit - savings
+$('#savings').find($('.deposit')).on('click',depositSavings);
+function depositSavings(){
+  var depositAmount = parseInt($('#savings').find($('.input')).val());
+  if(checkingBalance<0){
+    alert('Your checking account is overdrafted.  Submitting deposit to checking.');
+    overdraftCheck(depositAmount);
+  }
+  else{
+    savingsBalance = savingsBalance + depositAmount;
+    $('#savings').find($('.balance')).html('$' + savingsBalance);
+    alert('Thank you.  GNB loves you!');
+  }
+};
+
+// Withdraw - savings
+$('#savings').find($('.withdraw')).on('click',withdrawSavings);
+function withdrawSavings(){
+  var withdrawAmount = parseInt($('#savings').find($('.input')).val());
+  savingsBalance = savingsBalance - withdrawAmount;
+  if(savingsBalance<0){
+    alert('Insufficient funds.  Withdrawing from checking account.');
+    checkingBalance = checkingBalance + savingsBalance;
+    if(checkingBalance<0){
+      savingsBalance=0;
+      overdraftApproval();
+    }
+    else if(checkingBalance=0){
+      savingsBalance=0;
+      $('#checking').find($('.balance')).html('$' + checkingBalance);
+      $('#savings').find($('.balance')).html('$' + savingsBalance);
+      $('#checking').find($('.balance')).css('background-color','red');
+      alert('Thank you.  Please take your cash.  And remember, GNB loves you!');
+    }
+    else{
+      savingsBalance=0;
+      $('#checking').find($('.balance')).html('$' + checkingBalance);
+      $('#savings').find($('.balance')).html('$' + savingsBalance);
+      alert('Thank you.  Please take your cash.  And remember, GNB loves you!');
+    }
+  }
+  else{
+    savingsBalance=0;
+    $('#savings').find($('.balance')).html('$' + savingsBalance);
+    alert('Thank you.  Please take your cash.  And remember, GNB loves you!');
+  }
+};
+
```

Comment:
I like the way and the logic you'ved used to break down your program into functions.